15_1_Starting_the_Regtest.md

```diff
@@ -0,0 +1,31 @@
+# 15.1: Starting the Regtest
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+This document explains how to build a Regtest (Regression Test) by hand to be able to develop new applications without the need to interact with other peers and blocks.
+Bitcoin Core’s regression test mode (regtest mode) lets you instantly create a brand-new private block chain with the same basic rules as testnet—but one major difference: you choose when to create new blocks, so you have complete control over the environment.
```

Comment:
I prefer the word `local` over `private`. A private blockchain has connotations which do not apply to regtest.